web/src/components/CityContentFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import styled from 'styled-components'
+
+import { DISCLAIMER_ROUTE, LICENSES_ROUTE, pathnameFromRouteInformation } from 'api-client'
+
+import buildConfig from '../constants/buildConfig'
+import CleanLink from './CleanLink'
+import Footer from './Footer'
+
+const SidebarFooterContainer = styled.div`
+  width: 100%;
+  margin: -10px 12px 0 23px;
+  > * {
+    color: ${props => props.theme.colors.textColor};
+    height: 48px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    padding-right: 5.5px; /* the text is centered, the lines between are not */
```

Comment:
if you remove the margin above you should also be able to remove that padding

![image](https://user-images.githubusercontent.com/37902063/195550639-222feec8-3e86-493b-8905-6fb071602169.png)